packages/server-core/src/setting/client-setting/client-setting.seed.ts

```diff
@@ -99,7 +100,8 @@ export async function seed(knex: Knex): Promise<void> {
         if (!item.appleTouchIcon) {
```

Comment:
There should be a separate if condition for privacyPolicy.

Reply:
Sorry, I missed it. I have added this now.